src/nv_ingest/modules/transforms/nemo_doc_splitter.py

```diff
@@ -188,14 +140,11 @@ def split_and_forward(message: ControlMessage):
                         "DocumentSplitter only works with text documents but one or more " "'content' values are None."
                     )
 
-                units = _split_into_units(content, split_by)
-                text_splits = _concatenate_units(
-                    units,
-                    split_length,
-                    split_overlap,
-                    max_character_length=max_character_length,
-                )
-                split_docs.extend(_build_split_documents(row, text_splits, sentence_window_size=sentence_window_size))
+                os.environ['TOKENIZERS_PARALLELISM'] = "False"
```

Comment:
This should probably only be done once, and it has potential ramifications across the pipeline, because at some point tokenizers may be used somewhere else. Lets either set this at stage construction, or maybe even at pipeline level construction.

Reply:
I think this is actually safe to take out completely, forgot to remove it earlier